docs/kb/api-version-3/campaigns.md

```diff
@@ -79,6 +79,7 @@ For a complete list of fields involved in user operations, see the [Campaign](..
 | **Operation** | **URL Format**                             | **Required Parameters** | **Description**  |
 | ------------- | ------------------------------------------ | ----------------------- | -----------------|
 | `read` | `/api/campaign/version/3/do/read/id/`**_`<id>`_**`?...` | `user_key, api_key, id` | Returns the data for the campaign specified by `<id>`. `<id>` is the Pardot ID of the target campaign. |
+| `read` | `/api/campaign/version/3/do/read/fid/`**_`<fid>`_**`?...` | `user_key, api_key, fid` | Returns the data for the campaign specified by `<fid>`. `<fid>` is the Salesforce ID of the target campaign. |
```

Comment:
Question, it's called 'fid' here but 'crm_fid' in the object field reference page. I think that could be confusing, thoughts?